appveyor.yml

```diff
@@ -12,6 +12,7 @@ environment:
 
 install:
   - ps: Install-Product node $env:nodejs_version
+  - npm install --save fs-extra
```

Comment:
So, this is adding fs-extra to the package.json

What exactly is this needed for? And if it's needed, it should probably be added to the package.json rather than installed manually for this, and if it's installed manually for this ... then it shouldn't really use --save.